radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/HighQC.java

```diff
@@ -103,7 +103,8 @@
 	private final TimeoutCertificate highestTC;
 
 	@JsonCreator
-	private static HighQC serializerCreate(
+	@VisibleForTesting
```

Comment:
What does it mean?

Reply:
I need to access method from test (sometimes from other package) and for this purpose, I'm making it visible.